modules/afmongodb/libmongo-client

```diff
@@ -1 +1 @@
-Subproject commit 394a33880dfcadc46bcfba8334209bcb55b42106
```

Comment:
It can be good, but are you sure that you want to update libmongo-client in this PR?